controllers/cluster_controller.go

```diff
@@ -258,9 +259,10 @@ func (r *ClusterReconciler) listChildren(ctx context.Context, cluster *clusterv1
 	}
 
 	lists := map[string]runtime.Object{
-		"MachineDeployment": machineDeployments,
-		"MachineSet":        machineSets,
-		"Machine":           machines,
+		"MachineDeployment":   machineDeployments,
+		"MachineSet":          machineSets,
+		"Machine":             machines,
+		"MachineControlPlane": controlplanes,
 	}
 	for name, list := range lists {
```

Comment:
Map key sort order is not guaranteed to be consistent

Reply:
Argh, true. Switched to a slice instead